### ISSUE TITLE
Error when trying to publish a champer

### ISSUE THREAD

**Issue by Genboro** [BEEmod/BEE2.4#1476]
Hi,

I get this error when trying to publish the map, I used Portal 1 style with this one.
I use the latest version of Bee2 mod.

loadportals: couldn't read c:\steam\steamapps\common\portal 2\sdk_content\maps\1612519230.prt

I have no idea to fix it and I have a zero experience with hammer editor.
I've tried to search the similar error on internet and I found out that it means something like a leaked map, I have no idea how to fix it without anything too advanced like hammer.
If you can help, I highly appreciate it.

**Comment by LautaroL20** [comment]
Does this error also happens to you if you try to build a chamber without any items on it?

**Comment by Genboro** [comment]
Hi, I've fixed it today when I competely deleted the Portal 2 folder, verified the cache on steam, which completely redownloaded the game. I tested the empty chamber with the downloaded game and it worked, after that I exported Bee2 to the game and tried it with the empty chamber again, without the error. However, I've accidentaly deleted the map I wanted to publish when I deleted the folder but I already recreated it so I hope I it will work for now.

Credits and thanks to discord users: limeline, zapidom, Teamspen210 and Frozendragon for helping me with it. I wouldn't have figured it out on my own.
